deepchem/utils/typing.py

```diff
@@ -16,6 +16,10 @@
 # The shape of a NumPy array
 Shape = Tuple[int, ...]
 
-# type of RDKit Mol object
+# type of RDKit object
 RDKitMol = Any
 RDKitAtom = Any
+
+# type of Pymatgen object
```

Comment:
+1 for these type labels! This is a nice way to get around the optional imports hampering typing :)